packages/compass-aggregations/src/components/settings/settings.jsx

```diff
@@ -154,19 +153,8 @@ class Settings extends PureComponent {
         <div className={classnames(styles.header)}>
           <div className={classnames(styles['header-title'])}>Settings</div>
           <div className={classnames(styles['header-btn-group'])}>
-            <TextButton
-              id="aggregations-settings-cancel"
-              className="btn btn-default btn-xs"
-              text="Cancel"
-              clickHandler={this.onCancelClicked.bind(this)}
-            />
-
-            <TextButton
-              id="aggregation-settings-apply"
-              className="btn btn-primary btn-xs"
-              text="Apply"
-              clickHandler={this.onApplyClicked.bind(this)}
-            />
+            <Button id="aggregations-settings-cancel" size="xsmall" onClick={this.onCancelClicked.bind(this)}>Cancel</Button>
+            <Button id="aggregation-settings-apply" size="xsmall" variant="primary" onClick={this.onApplyClicked.bind(this)}>Apply</Button>
```

Comment:
If we want we could update the `id` usage to `data-testid` and update the selector in the end to end tests and the `settings.spec.jsx` test.